fuuk/people/migrations/0003_refactor_grants.py

```diff
@@ -0,0 +1,93 @@
+                ('prefix', models.CharField(max_length=20, null=True, blank=True)),
+                ('suffix', models.CharField(max_length=20, null=True, blank=True)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Institution',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('name', models.CharField(max_length=200)),
+                ('name_en', models.CharField(max_length=200, null=True)),
+                ('name_cs', models.CharField(max_length=200, null=True)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='editor',
+            field=models.ForeignKey(blank=True, to=settings.AUTH_USER_MODEL, null=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_first_name',
+            field=models.CharField(max_length=50, null=True, blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_human',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, blank=True, to='people.Human', null=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_last_name',
+            field=models.CharField(max_length=50, null=True, blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_prefix',
+            field=models.CharField(max_length=20, null=True, blank=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_suffix',
+            field=models.CharField(max_length=20, null=True, blank=True),
+        ),
+        migrations.AddField(
+            model_name='grantcollaborator',
+            name='grant',
+            field=models.ForeignKey(related_name='collaborators', to='people.Grant'),
+        ),
+        migrations.AddField(
+            model_name='grantcollaborator',
+            name='human',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, blank=True, to='people.Human', null=True),
+        ),
+        migrations.AddField(
+            model_name='grantcollaborator',
+            name='institution',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, blank=True, to='people.Institution', null=True),
+        ),
+        migrations.AddField(
+            model_name='grant',
+            name='investigator_institution',
+            field=models.ForeignKey(blank=True, to='people.Institution', null=True),
+        ),
+        migrations.AlterUniqueTogether(
+            name='grantcollaborator',
+            unique_together=set([('human', 'grant')]),
+        ),
```

Comment:
The `ForeignKey` fields are separate by Django, but it doesn't seems to be any reason for that.